Makefile

```diff
@@ -13,12 +13,20 @@ PLATFORMS ?= linux_amd64 linux_arm64
 -include build/makelib/common.mk
 
 # ====================================================================================
-# Setup Images
+# Setup Output
 
-# even though this repo doesn't build images (note the no-op img.build target below),
-# some of the init is needed for the cross build container, e.g. setting BUILD_REGISTRY
--include build/makelib/image.mk
-img.build:
+# S3_BUCKET ?= crossplane.releases
```

Comment:
@jbw976 wasn't sure if we wanted to push these to the S3 bucket also? We do this for Crossplane but not for providers. Not sure if the helm chart is the differentiator.

Reply:
is this where the image lives? If so, can anyone push an image there?

Reply:
I believe this is where the Helm chart lives. I presume the image would live at https://hub.docker.com/u/crossplane.

Reply:
I think the helm chart actually lives at `crossplane.charts` (https://github.com/crossplane/oam-kubernetes-runtime/pull/90/files#diff-b67911656ef5d18c4ae36cb6741b7965R25), so I am going to wait for @jbw976 to confirm the purpose of this bucket 👍

Reply:
@hasheddan we do not want to push anything from this repo to a "releases" s3 bucket. We accidentally copy/paste did this for the providers when we bootstrapped them, then realized they were overwriting previous crossplane releases 🤯, see https://github.com/crossplane/crossplane/issues/1320 for more details on that mistake.

Publishing the image to dockerhub and the helm chart to crossplane.charts should be all we need for this repo :)